### ISSUE TITLE
PactJS is using lodash 4.17.11 which has security vulnerability CVE-2019-10744

### ISSUE THREAD

**Issue by ariesmcrae** [pact-foundation/pact-js#333]
## Software versions

* **OS**: e.g. Mac OS Mojave 10.14.5 (18F132)
* **Consumer Pact library**: 8.2.6
* **Node Version**: `v11.12.0`

## Expected behaviour

`npm audit --json` should show no CVE vulnerabilities

## Actual behaviour

Found security vulnerability 
https://npmjs.com/advisories/1065

on
`"@pact-foundation/pact": "^8.2.6"`
`"@pact-foundation/pact-node": "^8.6.2"`

## Steps to reproduce
Step 1) npm i -D @pact-foundation/pact@^8.2.6 @pact-foundation/pact-node@^8.6.2
Step 2) npm audit --json
Step 3) See json log file below

## Relevant log files

```json
{
  "actions": [
    {
      "action": "review",
      "module": "lodash",
      "resolves": [
        {
          "id": 1065,
          "path": "@pact-foundation/pact>@pact-foundation/pact-node>caporal>lodash",
          "dev": true,
          "optional": false,
          "bundled": false
        },
        {
          "id": 1065,
          "path": "@pact-foundation/pact-node>caporal>lodash",
          "dev": true,
          "optional": false,
          "bundled": false
        }
      ]
    }
  ],
  "advisories": {
    "1065": {
      "findings": [
        {
          "version": "4.17.11",
          "paths": [
            "@pact-foundation/pact>@pact-foundation/pact-node>caporal>lodash",
            "@pact-foundation/pact-node>caporal>lodash"
          ],
          "dev": true,
          "optional": false,
          "bundled": false
        }
      ],
      "id": 1065,
      "created": "2019-07-15T17:22:56.990Z",
      "updated": "2019-07-15T17:25:05.721Z",
      "deleted": null,
      "title": "Prototype Pollution",
      "found_by": {
        "link": "",
        "name": "Snyk Security Team"
      },
      "reported_by": {
        "link": "",
        "name": "Snyk Security Team"
      },
      "module_name": "lodash",
      "cves": [
        "CVE-2019-10744"
      ],
      "vulnerable_versions": "<4.17.12",
      "patched_versions": ">=4.17.12",
      "overview": "Versions of `lodash` before 4.17.12 are vulnerable to Prototype Pollution.  The function `defaultsDeep` allows a malicious user to modify the prototype of `Object` via `{constructor: {prototype: {...}}}` causing the addition or modification of an existing property that will exist on all objects.\n\n",
      "recommendation": "Update to version 4.17.12 or later.",
      "references": "- [Snyk Advisory](https://snyk.io/vuln/SNYK-JS-LODASH-450202)",
      "access": "public",
      "severity": "high",
      "cwe": "CWE-471",
      "metadata": {
        "module_type": "",
        "exploitability": 3,
        "affected_components": ""
      },
      "url": "https://npmjs.com/advisories/1065"
    }
  },
  "muted": [],
  "metadata": {
    "vulnerabilities": {
      "info": 0,
      "low": 0,
      "moderate": 0,
      "high": 2,
      "critical": 0
    },
    "dependencies": 139,
    "devDependencies": 876326,
    "optionalDependencies": 9381,
    "totalDependencies": 876465
  },
  "runId": "4c1c2145-7dcd-44ec-a2a7-12ed8cb6105e"
}

**Comment by TimothyJones** [comment]
Thanks! This is also reported here: https://github.com/pact-foundation/pact-node/issues/168

By coincidence, it will be resolved by https://github.com/pact-foundation/pact-node/pull/167 (which removes Caporal).

In the meantime, I don't believe there's a non-vulnerable version of Caporal that we can use yet.

**Comment by TimothyJones** [comment]
Fixed in `9.0.0`.
